src/Cart.js

```diff
@@ -37,15 +37,17 @@ class Cart extends Component {
 
 
     render () {
-        console.log("props", this.props)
-        console.log("state", this.state)
+        // console.log("props", this.props)
+        // console.log("state", this.state)
         const {lineItems} = this.props;
         const { onChange, onUpdate, onDelete } = this;
         const TotalAmount = parseFloat(lineItems.reduce((acc, item) => {
             acc += item.quantity * item.price
             return acc
         }, 0)).toFixed(2);
-        console.log('isLogin: ', this.props.isLogin)
+        const disableCheckout = TotalAmount !== 0;
```

Comment:
There are a couple of changes to make here, as the way it is set up, the check out button is always disabled.

1. TotalAmount is going to get returned as a string, so it will need to be coerced to a number.
2. You have it set up so the cart will be disabled when total amount is not 0. Should disable when it is 0.
3. TotalAmount should camelCased as totalAmount.

Please change the name of the total amount variable to totalAmount, then set disable checkout as:

const disableCheckout = +totalAmount === 0

Reply:
@5t4mp5 correction has been made. TotalAmount changed to totalAmount, and typeof totalAmount is now a number. the checkout is now disable when totalAmount === 0.